pkg/apis/work/v1alpha2/well_known_constants.go

```diff
@@ -58,6 +58,10 @@ const (
 	// - Manifest in Work object: describes the name of ClusterResourceBinding which the manifest derived from.
 	ClusterResourceBindingAnnotationKey = "clusterresourcebinding.karmada.io/name"
 
+	// KarmadaWorkloadGenerationAnnotationKey records the generation of karmada workload to member workload.
+	// This annotation is helpful to generate observedGeneration for karmada workload.
+	KarmadaWorkloadGenerationAnnotationKey = "workload.karmada.io/generation"
```

Comment:
The resource wrapped in work is not necessarily a workload resource. It may be more appropriate to name it resourcetemplate.

Reply:
+1

Reply:
fixed

Reply:
I have a question. Does this annotation record the resource template generation of the host cluster or the workload generation of the member cluster?

Reply:
@zhy76 this annotation records the generation of the resource template in Karmada APIServer.
You refer to the latest comments.